themes/default/input.css

```diff
@@ -22,6 +22,11 @@
   border-color: #CC0302;
 }
 
+select:disabled {
```

Comment:
I actually thought we wanted this to apply to all select inputs but maybe we don't?